### PR TITLE
Issue #185: Display Title, Description and Domain in link preview for a tip

### PR DESCRIPTION
**Pull request by stanislav-slavov**

### CODE REVIEW

File: src/components/tipRecords/TipRecord.vue [hunk 1]
```diff
@@ -20,9 +20,20 @@
...
           <img v-bind:src="tip.preview.image" class="mr-2">
-          <span>
-            {{tipText}}
-          </span>
+          <div class="tip__article__content">
+            <h2 class="title text-ellipsis" 
+              :title="tipPreviewTitle">
+              {{tipPreviewTitle}}
+            </h2>
+            <div class="description"
```
Comment (venimus) [inline src/components/tipRecords/TipRecord.vue]: do we need text-ellipsis here too?

Comment (stanislav-slavov) [reply src/components/tipRecords/TipRecord.vue]: The description is being cut off with ellipsis after specific number of rows

**Comment by mradkov** [comment]
Can you resolve the conflicts @stanislav-slavov ?

**Comment by stanislav-slavov** [comment]
> Can you resolve the conflicts @stanislav-slavov ?

Done.

**Pull request merged**